### PR TITLE
Wix: Static Pages

### PR DESCRIPTION
**Pull request by akirk**
This adds preliminary static page parsing.

### Testing instructions
- Build the extension with `npm run build` and load it into your browser.
- Load your Wix dashboard
- Open the extension window and click export
- The WXR should contain your static pages.

### CODE REVIEW

**Comment by akirk** [comment]
Thanks to #23 having being merged, I've made some progress on importing menus. Unfortunately they are currently invalid since we need to interlink the post ids (which are currently all `0`).

Screenshot 2021-03-11 at 17 55 40

**Comment by pento** [comment]
> Unfortunately they are currently invalid since we need to interlink the post ids (which are currently all `0`).

Oh, the post ID generating was something I didn't end up finishing, as I hadn't really decided how it should behave. The current behaviour (requiring the app to keep track of post IDs) isn't great.

I'm currently leaning towards having a flag when you create the WXR driver to tell the driver to generate all post IDs itself, and ignore any passed IDs. I'm happy to explicitly not support mixing generated post IDs and passed post IDs.

**Comment by akirk** [comment]
I'm fine with the current behavior since the app needs to have the ability to line up an internal id mapping. Even for menus attached to posts, I first need to register the taxonomy terms which already need to point to the right post ids.

**Comment by akirk** [comment]
I created a very simple [IdFactory](https://github.com/pento/free-as-in-speech/blob/2c59b761a0d43c4c31c9fb7ec06bc1ca3a254aa4/source/utils/idfactory.js) which means that menus are now generated and work:

Wix frontend:
Screenshot 2021-03-15 at 15 35 00

Wix Editor:
Screenshot 2021-03-15 at 15 38 03

wp-admin:
Screenshot 2021-03-15 at 15 37 09

WordPress frontend:
Screenshot 2021-03-15 at 15 36 58

I have not yet found out how to automatically activate them upon import. Maybe it's not possible. At the moment you need to either assign the menu via the Customizer or Appearance > Menu > tick the Primary Menu checkbox and click Save.

**Comment by akirk** [comment]
I've refactored this so that it is ready for review. I'd see this mostly as an iteration that we'll need to continue to improve, so I'd like to merge this status quo to move forward in another PR.

File: package.json [hunk 1]
```diff
@@ -29,6 +29,9 @@
 		"@wordpress/wxr": "file:packages/wxr",
 		"cheerio": "1.0.0-rc.5",
 		"dayjs": "1.10.4",
+		"file-saver": "2.0.5",
```
Comment (pento) [inline package.json]: I think this was accidentally included in a rebase, perhaps. We don't use that package any more.

**Review by pento — APPROVED** [review]
> I've refactored this so that it is ready for review. I'd see this mostly as an iteration that we'll need to continue to improve, so I'd like to merge this status quo to move forward in another PR.

Agreed, let's run with it. One minor comment, but otherwise, let's get it merged and move onto the next thing.

**Comment by akirk** [comment]
status-update2

**Pull request merged**